hacking/build_library/build_ansible/command_plugins/plugin_formatter.py

```diff
@@ -185,7 +185,7 @@ def get_plugin_info(module_dir, limit_to=None, verbose=False):
 
         # Do not list blacklisted modules
```

Comment:
Here `blacklisted` is still written (@acozine commented on that and the comment was resolved, but it wasn't fixed).